src/app/components/Picture/index.module.scss

```diff
@@ -0,0 +1,5 @@
+.picture {
+    img {
+        max-width: 100%;
```

Comment:
For this styling the module.scss is not needed. just give the img the className `tw-max-w-full`

Reply:
removed